### PR TITLE
feat(card-link): update v2

### LINKED ISSUE #10751: [Component] Card link
### Reviews
- Component review [log notes](https://ibm.ent.box.com/notes/1196282090517)

### Design change log

- [ ] Add border around card
- [ ] Move arrow to right corner
- [ ] Merge with card component

### Spec 
- _Final spec file coming soon_
- [WIP spec](https://www.figma.com/file/sC8r1WIBS4TCczO0KvY6Tf/Card-link-v2?type=design&node-id=40%3A52408&mode=design&t=otATHPYnVwFpk9nO-1)

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

Closes #10751 

### Description

v2 updates

### Changelog

**New**

- add link as an option to `card`
- confirm `disabled` works

### CODE REVIEW

File: packages/styles/scss/components/card/_card.scss [hunk 1]
```diff
@@ -433,6 +433,18 @@
         }
       }
     }
+
+    &[link] {
```
Comment (IgnacioBecerra) [inline packages/styles/scss/components/card/_card.scss:437]: Should this be its own thing instead?
```
  :host(#{$c4d-prefix}-card[link])
```

File: packages/web-components/src/components/card/__stories__/card.stories.ts [hunk 2]
```diff
@@ -360,6 +363,87 @@ Static.story = {
...
+    const card = document.querySelector('c4d-card') as any;
+    const duration = card?.videoTitle?.match(/\((.*)\)/)?.pop();
+
+    if (!customVideoTitle) {
+      videoCopy = card?.videoTitle;
+    } else {
+      videoCopy = customVideoTitle;
+    }
+
+    console.log(card.querySelector('c4d-card-footer'), duration);
```
Comment (IgnacioBecerra) [inline packages/web-components/src/components/card/__stories__/card.stories.ts:382]:
```suggestion
```

**Review by IgnacioBecerra — COMMENTED** [review]
Nice!! Glad to see the card-link component finally getting replaced 🙌 .
Just a couple things:

File: packages/styles/scss/components/card/_card.scss (hunk 1 above)
Comment (ariellalgilmore) [reply packages/styles/scss/components/card/_card.scss:437]: yes! will update

**Review by IgnacioBecerra — APPROVED** [review]
LGTM code-wise!

**Comment by oliviaflory** [comment]
Finally took a look and it's great!

**Pull request merged**